backend/config/settings/base.py

```diff
@@ -786,21 +797,16 @@ def set_logging(level):
     AUTH_LDAP_SERVER_URI = env.str("AUTH_LDAP_SERVER_URI", None)
     AUTH_LDAP_BIND_DN = env.str("AUTH_LDAP_BIND_DN", None)
     AUTH_LDAP_BIND_PASSWORD = env.str("AUTH_LDAP_BIND_PASSWORD", None)
-    AUTH_LDAP_START_TLS = env.str("AUTH_LDAP_START_TLS", False)
```

Comment:
These LDAP lines are required and shouldn't be removed. Removing them would cause people not to login in Berlin instance.

Reply:
By that I mean all the subsequent lines as well.

Reply:
Corrected.